### PR TITLE
enable block limit after reordering and support the same for Chain health

### PR DESCRIPTION
**Pull request by igor-aptos**

### CODE REVIEW

**Comment by igor-aptos** [comment]
@sitalkedia @manudhundi @bchocho - this is ready for review

File: config/src/config/consensus_config.rs [hunk 1]
```diff
@@ -245,35 +260,39 @@ impl Default for ConsensusConfig {
...
+                    backoff_if_below_participating_voting_power_percentage: 72,
                     // in practice, latencies and delay make it such that ~2 blocks/s is max,
                     // meaning that most aggressively we limit to ~500 TPS
                     // For transactions that are more expensive than that, we should
                     // instead rely on max gas per block to limit latency
                     max_sending_block_txns_override: 250,
-                    max_sending_block_bytes_override: 1024 * 1024 + BATCH_PADDING_BYTES as u64,
+                    max_sending_block_bytes_override: MIN_BLOCK_BYTES_OVERRIDE,
                     backoff_proposal_delay_ms: 300,
+                    max_txns_from_block_to_execute: None,
```
Comment (manudhundi) [inline config/src/config/consensus_config.rs:295]: All the max_txns_from_block_to_execute are None in ChainHealthBackoffValues.
Don't we want to consider stricter ChainHealthBackoffValues ?

Comment (igor-aptos) [reply config/src/config/consensus_config.rs:295]: oh. I wrote a separate PR assuming they cannot go in the same release.

but the support you built there is what both use, so they can go. let me just add those changes in this PR...

thanks for the callout :)

Comment (igor-aptos) [reply config/src/config/consensus_config.rs]: updated

**Review by manudhundi — APPROVED** [review]

**Review by sitalkedia — APPROVED** [review]
LGTM

**Pull request merged**